src/sentry/workflow_engine/processors/workflow.py

```diff
@@ -258,7 +253,12 @@ def process_workflows(event_data: WorkflowEventData) -> set[Workflow]:
     with sentry_sdk.start_span(
         op="workflow_engine.process_workflows.evaluate_workflows_action_filters"
     ):
-        actions = evaluate_workflows_action_filters(triggered_workflows, event_data)
```

Comment:
🤔 going to try a spike to see if we can refactor this a little to get the dcga's with the workflow id in the values list. that will allow us to fix the log / metric below for "all" and then reduce the amount of work to create the workflow id association as well. 🤞